Scripts/addImagesToDocumentation.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Move to project root directory
+scriptDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$scriptDir/../"
+
+PATH_TO_SNAPSHOTS="Tests/StreamChatUITests/UISDKdocumentation/__Snapshots__"
+
+for UI_SNAPSHOT in ${PATH_TO_SNAPSHOTS}/*;do
+    STRIPPED_PATH=`basename $UI_SNAPSHOT`
+    COMPONENT_NAME=${STRIPPED_PATH%_*_*}
+    
+    DOCUMENTATION_FILE=`find Documentation/Sources/StreamChatUI -name "_$COMPONENT_NAME.md"`
+    # Let's use just light variation of the screenshot, we can support dark mode later.
+    FINAL_SNAPSHOT=`ls $UI_SNAPSHOT | grep light`
+    tail -1 "$DOCUMENTATION_FILE" | grep "$FINAL_SNAPSHOT"
```

Comment:
I actually think there's no point in generating the dark version 🤔 Or does swift-docs support light/dark mode? Because the point of this is to only verify the component hierarchy, and light/dark mode doesn't influence anything on the hierarchy. So my suggestion is for generating the hierarchy snapshot only for light mode.

Reply:
Hmm I was thinking that we might wanna support dark mode in our documentation, because you know how handy this comes. But there is problem with markdown and I am not quite sure how we can handle this in docusaurus, this needs some research as well... I will check if we can support dark/light mode with docusaurus and try to figure out what to do with this...

Reply:
I think docussarus supports dark/Light mode. But even if it does, I think change the image to dark or light is not really the point of this I guess? We can just put a static background colour of the component hierarchy? 🤔 Lets discuss this with the team on Monday